src/app/trainees/trainee-list-paginator/trainee-list-paginator.component.spec.ts

```diff
@@ -4,20 +4,28 @@ import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 import { MatPaginatorModule, PageEvent } from "@angular/material/paginator";
 import { RouterTestingModule } from "@angular/router/testing";
 import { NgxsModule, Store } from "@ngxs/store";
-import { of } from "rxjs";
-import {
-  GetTrainees,
-  PaginateTrainees,
-  UpdateTraineesRoute
-} from "../state/trainees.actions";
+import { Observable, of } from "rxjs";
+import { TraineeService } from "../../core/trainee/trainee.service";
+import { GetTrainees, PaginateTrainees } from "../state/trainees.actions";
 import { TraineesState } from "../state/trainees.state";
 
 import { TraineeListPaginatorComponent } from "./trainee-list-paginator.component";
 
+export class MockTraineeService {
```

Comment:
no need for export unless re-used in different tests? Could be in separate file if its going to be re-used and imported?

Reply:
ah yes very good catch, cheers. using the export keyword has become bit of a habit oops. i think best to keep mocks within the spec itself otherwise the tests can become brittle by creating a dependancy.

pr updated